deploy/install.sh

```
@@ -59,65 +51,76 @@ function installing_onboarder() {
-
-if [[ "$answer" =~ [yY](es)* ]]; then
-  echo -e "\e[1m\e[32mPartners onboarded successfully.\e[0m"
-else
-  echo -e "\e[1m\e[31mPartner onboarding steps are pending. Please complete the configuration steps for onboarding partner.\e[0m"
-fi
+      --set onboarding.secrets.s3.s3-user-secret="$s3_user_secret" \
+      --set onboarding.configmaps.s3.s3-host="$s3_url" \
+      --set onboarding.configmaps.s3.s3-user-key="$s3_user_key" \
+      --set onboarding.configmaps.s3.s3-region="$s3_region" \
+      --set onboarding.configmaps.s3.s3-bucket-name="$s3_bucket" \
+      $ENABLE_INSECURE \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)